HangMan/src/hangman.rs

```diff
@@ -0,0 +1,139 @@
+use std::vec;
+use dioxus::prelude::*;
+
+//component for managing the image of the hangman
+#[inline_props]
+pub fn Hangman(cx:Scope, stage: usize, reset_image: bool) -> Element {
+    
+    let css_status = use_ref(cx, || {vec![true;11]});    
+
+    //changes the hidden status of the next image in queque when the user makes a guess
+    use_memo(cx, stage, |stage| {
+        
+        css_status.with_mut(|current_list| {
+            if stage >= current_list.len() {
+                panic!("SHOULDNT ALLOW THE USER TO MAKE MORE THAN 10 GUESSES");
+            } else {
+                current_list[stage] = false;
+            }
+        })   
+    });
+
+    use_memo(cx, reset_image, |_| {
+        css_status.with_mut(|cur| {
+            *cur = vec![true;11];
+        })
+    });
+
+    render!(rsx!(
+        link { rel: "stylesheet", href: "HangMan/input.css"}
+        div {
```

Comment:
I don't see parts of a hangman appear when I get guesses wrong

Reply:
Oh I was running the example from the wrong path. If you run it inside of the hangmain folder it fails, but from outside the hangman folder this works:
```
cargo run --package HangMan
```